configuration/parse.go

```diff
@@ -174,7 +173,6 @@ Options:
 
 	err := loadConfigFromFile(ConfigLocation, &config)
 	if err != nil {
-		fmt.Println(err)
```

Comment:
should probably log the error here and above, big difference between a missing file and a malformed file

Reply:
or add logging to the error site

Reply:
In : ba07c9d

Reply:
Same thing for `loadIQConfigFromFile`

Reply:
In: 23492bf

Reply:
Actually 1 more nit to pick... `Unable to read file` might be a little confusing without context, might want to mention `Unable to read (IQ|Ossindex) config file` (possibly with the path as a logged field?)